Software/app/freefall_lorawan/sys_app.c

```diff
@@ -0,0 +1,378 @@
+/**
+  ******************************************************************************
+  * @file    sys_app.c
+  * @author  MCD Application Team
+  * @brief   Initializes HW and SW system entities (not related to the radio)
+  ******************************************************************************
+  * @attention
+  *
+  * <h2><center>&copy; Copyright (c) 2020 STMicroelectronics.
+  * All rights reserved.</center></h2>
+  *
+  * This software component is licensed by ST under Ultimate Liberty license
+  * SLA0044, the "License"; You may not use this file except in compliance with
+  * the License. You may obtain a copy of the License at:
+  *                             www.st.com/SLA0044
+  *
+  ******************************************************************************
+  */
+
+#include <stdio.h>
+#include "app.h"
+#include "sys_app.h"
+#include "stm32_seq.h"
+#include "stm32_systime.h"
+#include "stm32_lpm.h"
+#include "GNSE_rtc.h"
+
+#define MAX_TS_SIZE (int)16
+
+/**
+  * @brief  Set all pins such to minimized consumption (necessary for some STM32 families)
+  * @param none
+  * @return None
+  */
+static void Gpio_PreInit(void);
+
+/**
+  * @brief  Initialises the system for debugging or low power mode depending on DEBUGGER_ON
+  * @param none
+  * @return None
+  */
+static void DBG_Init(void);
+/**
+
+  * @brief Returns sec and msec based on the systime in use
+  * @param none
+  * @return  none
+  */
+static void TimestampNow(uint8_t *buff, uint16_t *size);
+
+/**
+  * @brief  it calls UTIL_ADV_TRACE_VSNPRINTF
+  */
+static void tiny_snprintf_like(char *buf, uint32_t maxsize, const char *strFormat, ...);
+
+ACC_op_result_t Accelerometer_Init(void)
+{
+    /* Set load switch */
+    GNSE_BSP_LS_Init(LOAD_SWITCH_SENSORS);
+    GNSE_BSP_LS_On(LOAD_SWITCH_SENSORS);
+    HAL_Delay(100);
+
+    GNSE_BSP_Sensor_I2C1_Init();
+    HAL_Delay(100);

```

Comment:
This is not related to Accelerometer_Init(), please move this part exactly as it is to https://github.com/TheThingsIndustries/generic-node-se/blob/94500111cde0040a24c9e71e5b88fd773aed7bc3/Software/app/basic_lorawan/sys_app.c#L81

Here is why it's not related:
`GNSE_BSP_LS_Init` and `GNSE_BSP_LS_On` power gates all the sensors.
`GNSE_BSP_Sensor_I2C1_Init` initializes the I2C bus connected to all the sensors.

What if you want to add another sensor to this freefall app, would you use this Init for as well?